common/config/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+project(cgaconfig)
+
+find_package(CUDA 9.0 QUIET REQUIRED)
+
+add_library(${PROJECT_NAME} INTERFACE)
+configure_file("cga_config.hpp.in" "include/claragenomics/cga_config.hpp")
```

Comment:
The idea was to have just one global header with all needed global configuration, where also the CUDA version marcos live (but also other configuration in the future).
Right now `configure_file` is not required, but if this is a central configuration file, it probably will be needed in the near future

Reply:
Since the amount of global configuration should be minimal I would not start to modularize it any further.

Reply:
Agreed on the modularization, can leave it at the `claragenomics` level, but I think the `configure_file` setup maybe overkill right now. since it should be quite minimal, I am hoping it wouldn't require any environment/setup specific configs in it.

Reply:
ok, done.